charts/fluent-operator/README.md

```diff
@@ -0,0 +1,60 @@
+# Fluent Operator Helm Chart
+
+[Fluent Operator](https://github.com/fluent/fluent-operator) provides great flexibility in building a logging layer based on Fluent Bit and Fluentd.
+
```

Comment:
I think you need to call out the issue with Helm not upgrading CRDs: https://helm.sh/docs/chart_best_practices/custom_resource_definitions/

This is a super common cause of issues as quite a few people are unaware of it. We should indicate how to do an upgrade properly including CRDs.

Reply:
Agree. @wenchajun you'll need to add a link to https://helm.sh/docs/chart_best_practices/custom_resource_definitions/ saying:
Fluent Operator CRDs will be installed by default when running a helm install for the chart. But if the CRD already exists, it will be skipped with a warning. So make sure you install the CRDs by yourself if you upgrade your Fluent Operator version.

Reply:
I'd suggest adding how to do it, e.g. something like https://docs.couchbase.com/operator/current/helm-managing-guide.html#upgrade-a-deployment

Pull the chart, grab the CRDs then `kubectl replace ...` them.
Otherwise if you're not careful someone will do it incorrectly and trigger an issue here.